extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -733,7 +737,7 @@ export class AmpAnalytics extends AMP.BaseElement {
    */
   expandTemplateWithUrlParams_(spec, expansionOptions) {
     return this.variableService_
-      .expandTemplate(spec, expansionOptions)
+      .expandTemplate(spec, expansionOptions, this.element)
```

Comment:
Since we pass the binding info to `expandUrlAsync`, do we want to pass the same to `expandTemplate`?

Reply:
I think this only matters for `RESOURCE_TIMING` as the way `expandTemplate` is now written it will try to get the bindings from the element if not passed in. If you prefer to explicitly pass it the bindings everywhere, I am happy to do so.